projects/core/src/occ/services/occ-endpoints.service.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('OccEndpointsService', () => {
   });
 
   it('should return base endpoint', () => {
```

Comment:
You can delete those cases they are duplicates.

Reply:
Done